### PR TITLE
feat: (Product Partnerships) aws natgateway

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New AWS NatGateway quickstart. Part of Amazon VPC, uses cloudwatch metrics.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/amazon-vpc-nat-gateway/config.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  ## Monitor your NAT Gateway in New Relic's observability platform
+  Our NAT Gateway integration reports data from your NAT Gateway service to your New Relic account using AWS CloudWatch Metric Streams.
+  ## Get started!
+  This integration uses [AWS CloudWatch Metric streams](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/connect/aws-metric-stream/) to get NAT Gateway traffic data into New Relic.
+  ## About this integration
+  With this integration you can monitor key traffic information about your NAT Gateway.
+  - Track your connection established rate
+  - View idle connection timeouts
+  - Track Byte traffic
+  - Monitor your active connections over timeouts
```
Comment (seemantk) [inline quickstarts/aws/amazon-vpc-nat-gateway/config.yml:12]: The bit about "over timeouts" is a little unclear to me

**Review by seemantk — COMMENTED** [review]
this looks great @jcountsNR

File: quickstarts/aws/amazon-vpc-nat-gateway/config.yml (hunk 1 above)
Comment (jcountsNR) [reply quickstarts/aws/amazon-vpc-nat-gateway/config.yml:12]: should be over time, not timeouts. I think VSCode auto completed that one for me.

**Review by seemantk — APPROVED** [review]

File: quickstarts/aws/amazon-vpc-nat-gateway/config.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-nat-gateway-monitoring-integration/
+keywords:
+  - aws
+  - amazon
+  - nat gateway
+  - metrics
+  - vpc
+  - network
+installPlans:
+  - aws-cloudwatch-metric-streams
```
Comment (josephgregoryii) [inline quickstarts/aws/amazon-vpc-nat-gateway/config.yml:35]: We're currently in a deprecation period for `installPlans` - we are trying to transition into `Data Sources` for our quickstart installs. We would want to change `installPlans` to `dataSourceIds`

The docs can on the install plan deprecation can be viewed [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#install-plans-deprecated).

Shortly below are the docs for Data Sources. 

We have a data source equivalent [here](https://github.com/newrelic/newrelic-quickstarts/blob/main/data-sources/amazon-cloudwatch-metric-streams/config.yml) which you can use.

**Review by josephgregoryii — CHANGES_REQUESTED** [review]
Hey @jcountsNR 

Quickstart looks great, just one request about using a `Data Source` instead of an `Install Plan`

File: quickstarts/aws/amazon-vpc-nat-gateway/config.yml (hunk 2 above)
Comment (aswanson-nr) [inline quickstarts/aws/amazon-vpc-nat-gateway/config.yml:35]: We actually have a data source for this! 
```suggestion
dataSourceIds:
  - amazon-cloudwatch-metric-streams
```

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**